specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json

```diff
@@ -5164,6 +2857,10 @@
         "errorDetail": {
           "$ref": "#/definitions/ErrorDetail",
           "description": "Error Detail in case the status is non-success."
+        },
+        "additionalDetail": {
```

Comment:
Please remove this api breaking change.

Reply:
This is a non-editable, system generated property. Marked this as read-only.